entrypoint.sh

```diff
@@ -11,6 +11,16 @@ PG_CONFIG_DIR=/etc/pgbouncer
 if [ -n "$DATABASE_URL" ]; then
   # Thanks to https://stackoverflow.com/a/17287984/146289
 
+  # Check that begins with valid postgres:// prefix and has at max one '@' char
+  charcount=$(echo $DATABASE_URL | grep -o "@" | wc -l)
+  if [[ ! "$DATABASE_URL" =~ ^postgres:// ]]; then
```

Comment:
`postgresql` is also valid and some python library stopped supporting `postgres`

Reply:
I realize it's also possible to have formats like:
```
DATABASE_URL=postgresql+asyncpg://username:password@host:port/dtabase_name
```
so i think it's easiest to remove the check for connection string formatting for now